src/patternfly/upgrade-examples/AlertUpgradeExamples/examples/alert-upgrade-example-pf4-notification.hbs

```diff
@@ -0,0 +1,63 @@
+{{#> alert-upgrade-examples}}
+{{#> alert alert--modifier="pf-m-success" alert--attribute='aria-label="Success Notification"'}}
+  {{#> alert-icon alert-icon--success="true"}}
+  {{/alert-icon}}
+  {{#> alert-body alert-body--attribute='role="alert"'}}
+    {{#> alert-description}}
+			The server configuration changed.
+		{{/alert-description}}
+	{{/alert-body}}
+  {{#> alert-action}}
+		{{#> button btnClass="pf-c-button" btnAttributes='aria-label="Remove"'}}
+				<i class="fas fa-times"></i> 
+		{{/button}}
+    {{/alert-action}}
+{{/alert}}
+<br>
+{{#> alert alert--modifier="pf-m-danger" alert--attribute='aria-label="Danger Notification"'}}
+  {{#> alert-icon alert-icon--danger="true"}}
+  {{/alert-icon}}
+  {{#> alert-body alert-body--attribute='role="alert"'}}
+    {{#> alert-description}}
+			Failed to add server_abc.
+		{{/alert-description}}
+  {{/alert-body}}
+{{/alert}}
+<br>
+{{#> alert alert--modifier="pf-m-warning" alert--attribute='aria-label="Warning Notification"'}}
+  {{#> alert-icon alert-icon--warning="true"}}
+  {{/alert-icon}}
+  {{#> alert-body alert-body--attribute='role="alert"'}}
+    {{#> alert-description}}
+			The server configuration changed. 
+			{{#> alert-action alert-action--modifier="pf-c-button pf-m-link"}}
+        	Reload server.
+      {{/alert-action}}
+		{{/alert-description}}
+
+  {{/alert-body}}
+  {{#> alert-action}}
+		{{#> button btnClass="pf-c-button" btnAttributes='aria-label="Remove"'}}
```

Comment:
I wonder if the close button should be it's own element, and not listed as an action. Something like the close button in the [modal](https://pf-next.com/components/modalbox/examples/) or [popover](https://pf-next.com/components/Popover/examples/).

Reply:
Hmm, maybe? But in this case, there could possibly be any combination of actions, one of which could be a [X] close action. I think this is just debatable enough, that I would capture that as an issue for the Alert to be handled outside of this PR. It also might be good to let the design documentation land before making any more html/css updates to the component.

Reply:
@michael-coker @jgiardino so for now go ahead with the alert-action to encompass all combinations of actions? I can create an issue to address what we do in the future for alert actions?

Reply:
@christiemolloy yep!